neutrino-custom/lint.js

```diff
@@ -53,7 +53,53 @@ module.exports = (neutrino) => {
                     radix: 'off',
                     'react/forbid-prop-types': 'off',
                     'react/no-multi-comp': 'off',
+                    // Backport of:
```

Comment:
This is needed since otherwise the older AirBnb ESLint config `master` is using will complain that `componentWillUnmount()` needs to be put back after `componentDidCatch()` in `WatchedRepo.jsx` - since the older config doesn't know about the new `componentDidCatch` feature in React 16.x .